### ISSUE TITLE
Error: All components of query must be named

### ISSUE THREAD

**Issue by jnmaloof** [cscheid/rgithub#60]
rgithub had been working for me fine but now gives me an error when I try to get my repositories.  Bad interaction with another package?  Any advice?

```
> ctx <- create.github.context(access_token = Sys.getenv("GITHUB_TOKEN"),verbose=F)
> my.repos <- get.my.repositories(ctx)
Error: All components of query must be named

> sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.10.5 (Yosemite)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] github_0.9.7    devtools_1.10.0 plyr_1.8.3     

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.3     withr_1.0.1     digest_0.6.9    R6_2.1.2        jsonlite_0.9.19 git2r_0.13.1   
 [7] magrittr_1.5    httr_1.1.0      stringi_1.0-1   curl_0.9.5      rstudioapi_0.5  brew_1.0-6     
[13] rjson_0.2.15    tools_3.2.3     stringr_1.0.0   Rook_1.1-1      memoise_1.0.0  
> 
```

**Comment by cscheid** [comment]
Just FYI, I’m snowed under a lot of things and will only be able to get to this message in late Feb. Sorry about that.

**Comment by mfcovington** [comment]
@jnmaloof:

Can you confirm that `Sys.getenv("GITHUB_TOKEN")` returns the expected token in the environment that is failing?

Also, what do you see when `verbose=T`?

**Comment by jennybc** [comment]
I suspect that the `httr` version is a more likely explanation here.

@jnmaloof While you wait for someone to get to the bottom of it, you might trying downgrading `httr` one or even two versions to see if that helps.

**Comment by jnmaloof** [comment]
Thanks @jennybc .  Downgrading to `httr`  1.0.0 worked.

**Comment by jnmaloof** [comment]
@mfcovington yes, `Sys.getenv("GITHUB_TOKEN")` works as expected.

**Comment by jennybc** [comment]
I just finished updating another package for `httr v1.1.0` 😁.

**Comment by cscheid** [comment]
Thanks everyone for tracking it down.

Please don’t close this issue - now that we know it’s a problem with httr, I want to eventually fix it. It’ll just take a while. In the meantime, PRs always welcome.

**Comment by locklin** [comment]
`build_url(parse_url("https://engage.us.greenpeace.org/onlineactions/0Dz8eFnSfEGAkQBffzy_1A2?&utm_source=website&utm_medium=homepage&utm_campaign=website_donation_form_(header)&r=true&am=15"))`
is an example of something that barfs on the latest httr.

This isn't really something I actually want to crawl, so it's kind of OK that it barfs an exception, but now I have to handle the exception.

**Comment by cscheid** [comment]
Closed (wontfix) by #70.
